exersice_html/home.html

```diff
@@ -0,0 +1,285 @@
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+  <title>Shopka</title>
+</head>
+<body>
+  <header class="page-header">
+    <div class="container header-wrap">
+      <div class="header-left">
+        <h1>
+          <a href="#" class="logo">
+            <img class="logo-img" src="./image/logo.svg" alt="logo" />
+          </a>
+        </h1>
+      </div>
+      <div class="header-center">
+        <div class="menu">
+          <i class="fa fa-bars"></i>
+        </div>
+        <div class="search-group">
+          <span class="search-icon">
+            <i class="fa fa-search"></i>
+          </span>
+          <input class="search" placeholder="Search" />
+        </div>
+      </div>
+      <div class="header-right">
+        <a href="#" class="cart">
+          <i class="fa fa-shopping-cart"></i>
+          <span class="cart-amount">4</span>
+        </a>
+        <a href="#" class="sign-in">Sign in</a>
+        <a href="#" class="btn btn-primary">Sign up</a>
+      </div>
+    </div>
+    </div>
+  </header>
+  <main>
+      <section>
+        <div class="container banner">
+          <img class="banner-img" src="./image/banner.png" alt="banner" />
+        </div>
+      </section>
+      <section>
+        <div class="container">
+          <div class="toolbar">
+            <div class="sort">
+              <select class="sort-uselessFirst">
+                <option>Useless first</option>
+              </select>
+              <select class="sort-condition">
+                <option>Condition</option>
+              </select>
+              <select class="sort-delivery">
+                <option>Delivery option</option>
+              </select>
+            </div>
+            <div class="action">
+              <div class="action-group">
+                <div class="btn action-btn">
+                  <label for="show-all">Show all</label>
+                  <input type="radio" name="action" id="show-all"/>
+                </div>
+                <div class="btn action-btn">
+                  <label for="auction" class="btn action-btn">Auction</label>
+                  <input type="radio" name="action" id="auction"/>  
+                </div>
+                <div class="btn action-btn">
+                  <label for="buy-now" class="btn action-btn">Buy now</label>
+                  <input type="radio" name="action" id="buy-now"/> 
+                </div>  
+              </div>
+              <div class="type-show">
```

Comment:
I think this `div` should has the same class as above one. they are `action-group` too